### DISCUSSION TITLE
Clarification of Gene Integration Matrix values

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#1337] [General]
@rcorces 

Hi,

 What do the values in the Gene Integration Matrix represent? Is it based on  prediction of gene expression based on accessibility scores for the gene in question or is it strictly data derived from the scRNAseq data used in the cross platform integration.

Thanks!

**Comment by rcorces** [comment]
strictly data derived from sc/snRNA-seq data and the cross platform integration

**Reply by sylestiel** [reply]
Thanks!
